tests/eosio.rentbw_tests.cpp

```diff
@@ -350,37 +455,52 @@ struct rentbw_tester : eosio_system_tester
          ilog("after_reserve.net:                        ${x}", ("x", after_reserve.net));
          ilog("before_reserve.cpu:                       ${x}", ("x", before_reserve.cpu));
          ilog("after_reserve.cpu:                        ${x}", ("x", after_reserve.cpu));
-
-         csv->newRow() << last_block_time()
-                       << before_state.net.assumed_stake_weight
-                       << before_state.net.weight_ratio / double(rentbw_frac)
-                       << before_state.net.weight
-                       << before_receiver.net
-                       << after_receiver.net
-                       << after_receiver.net - before_receiver.net
-                       << before_payer.liquid - after_payer.liquid
-                       << before_reserve.net
-                       << after_reserve.net
-                       << before_reserve.cpu
-                       << after_reserve.cpu
-
-                       << get_state().cpu.weight
-                       << get_state().cpu.weight_ratio
-                       << get_state().cpu.assumed_stake_weight
-                       << get_state().cpu.initial_weight_ratio
-                       << get_state().cpu.target_weight_ratio
-                       << get_state().cpu.initial_timestamp.sec_since_epoch()
-                       << get_state().cpu.target_timestamp.sec_since_epoch()
-                       << get_state().cpu.exponent
-                       << get_state().cpu.decay_secs
-                       << get_state().cpu.min_price.to_string()
-                       << get_state().cpu.max_price.to_string()
-                       << get_state().cpu.utilization
-                       << get_state().cpu.adjusted_utilization
-                       << get_state().cpu.utilization_timestamp.sec_since_epoch()
-                       << get_state().cpu.fee
-
-                       << (max > 0 ? "rentbwexec" : "rentbw");
+   
+         csv->newRow() << last_block_time()           
+                      << before_state.net.assumed_stake_weight
+                      << before_state.net.weight_ratio / double(rentbw_frac)
+                      << before_state.net.weight
+                      << float(before_reserve.net / 10000.0)
+                      << float(after_reserve.net / 10000.0)
+                      << float(before_reserve.cpu / 10000.0)
+                      << float(after_reserve.cpu / 10000.0)
+                      << before_payer.liquid
+                      << net_frac
+                      << float(net_util / 10000.0)
+                      << float(net_fee / 10000.0)
+                      << cpu_frac
+                      << float(cpu_util / 10000.0)
+                      << float(cpu_fee / 10000.0)
+                      << float((before_payer.liquid - after_payer.liquid).get_amount() / 10000.0) 
```

Comment:
define a constant equal to 10000.0 instead of using `10000.0`

Reply:
is there a particular need to keep `before_reserve.net`, `after_reserve.net`, `net_util`, etc as integers and convert their values for output instead of dealing with floats with exact values? I just can't wrap my head around it

Reply:
Floats don't have exact values. nodeos and the reference contracts avoid use of floats except where necessary. The testing library pulls in the types that nodeos and the contracts use.

Reply:
thanks @tbfleming 
@deckb minor readability pick: replace `float(<asset> / asset_to_float_const)` with an inline function `to_human_readable(<asset>)` call. Where 
```
inline float to_human_readable(uint64 asset) { return asset / asset_to_float_const; }
```

Reply:
added.